xml/obs_best_practice_staging_workflow.xml

```diff
@@ -20,10 +20,10 @@
   <para>
     The maintenance of those packages can be made on a collaborative way via
     Build Service. As shown in the following diagram, the maintainer can create
-    a package and then she or any other developer can branch it (make a copy of
-    it), do some changes on its code and request those changes to be applied on
-    the original package. After that, the maintainer usually reviews the
-    request, chat with the developer in case it needs some fixes and end up
+    a package and then he or any other developer can branch it (make a copy of
```

Comment:
There was a phase in the 2000's where people used either "he or she" or just "she" to be gender-inclusive.
These days, the more common gender-inclusive form in English is just using "they".